x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/create_engine_logic.test.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { generatePath } from 'react-router-dom';
+
+import { CREATE_ENGINE_SUCCESS_MESSAGE } from './constants';
+import {
+  LogicMounter,
+  mockHttpValues,
+  mockKibanaValues,
+  mockFlashMessageHelpers,
+} from '../../../__mocks__';
+import { ENGINE_PATH } from '../../routes';
+import { formatApiName } from '../../utils/format_api_name';
+import { CreateEngineLogic, DEFAULT_LANGUAGE } from './create_engine_logic';
+
+describe('CreateEngineLogic', () => {
+  const { mount } = new LogicMounter(CreateEngineLogic);
+  const { http } = mockHttpValues;
+  const { navigateToUrl } = mockKibanaValues;
+  const { setQueuedSuccessMessage, flashAPIErrors } = mockFlashMessageHelpers;
+
+  const DEFAULT_VALUES = {
+    name: '',
+    rawName: '',
+    language: DEFAULT_LANGUAGE,
```

Comment:
It would personally help me to quickly grok/understand defaults better as static strings - since there are no performance implications with testing, can we change this to:
```suggestion
    language: 'Universal',
```